src/components/menu/menu.spec.ts

```diff
@@ -376,6 +383,38 @@ describe('MenuComponent', () => {
 
         expect(component.item1Activated).toHaveBeenCalledTimes(1);
     });
+
+    it('should not emit a click event when disabled is true.', async () => {
```

Comment:
Perhaps this should also mention that it checks for activate emit too? E.g. should not emit a click or activate event when disabled is true